### PR TITLE
refactor(installer): Remove Go references to dd-installer

### PR DESCRIPTION
**Pull request by BaptisteFoy**
### What does this PR do?
Removes Go references to the dd-installer as we are replacing it by dd-agent

### CODE REVIEW

File: cmd/installer/user_all.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ import (
 	"syscall"
 )
 
-func moveToDDInstaller() error {
-	grp, err := user.LookupGroup("dd-installer")
+func moveToDDAgent() error {
+	grp, err := user.LookupGroup("dd-agent")
```
Comment (raphaelgavache) [inline cmd/installer/user_all.go:19]: dd-agent group is looked twice now instead of once, see below

Comment (BaptisteFoy) [reply cmd/installer/user_all.go:19]: Good catch, fixed in [46e9146](https://github.com/DataDog/datadog-agent/pull/24919/commits/46e91465654e311c1c7835f504841dbd5f8c4526)

**Review by raphaelgavache — APPROVED** [review]
lgtm

**Pull request merged**